packages/spatial/src/physics/components/ColliderComponent.tsx

```diff
@@ -60,22 +83,104 @@ export const ColliderComponent = defineComponent({
     const physicsWorld = Physics.useWorld(entity)
     const triggerComponent = useOptionalComponent(entity, TriggerComponent)
     const hasCollider = useState(false)
+    const uuid = useComponent(entity, UUIDComponent)
+    const childMeshEntities = useChildrenWithComponents(entity, [MeshComponent])
+
+    const nestedCollidersState = getMutableState(NestedCollidersState)
```

Comment:
i think we need separating the user story here from the technical implementation.

Does the ColliderComponent specify a single collider, or potentially a hierarchy of colliders? I think it should be the former, and we can have a higher-level collider that handles which colliders exist/are enabled.

Reply:
I could see a new "collider manager" component working nicely, I personally feel like there was more conceptual iteration needed prior to starting this feature work...which I tried to kickstart